doc/devices/overview.rst

```diff
@@ -0,0 +1,65 @@
+Overview
+========
+
+Once the PennyLane-Qiskit plugin is installed, the following Qiskit devices
+can be accessed straightaway in PennyLane.
+
+Devices
+~~~~~~~
+
+Currently, there are three devices for different qiskit "providers":
+
+.. devicegalleryitem::
+    :name: 'qiskit.aer'
+    :description: Qiskit's staple simulator with great features such as noise models.
+    :link: aer.html
+
+.. devicegalleryitem::
+    :name: 'qiskit.basicaer'
+    :description: A simplified version of the Aer device, which requires fewer dependencies.
+    :link: basicaer.html
+
+.. devicegalleryitem::
+    :name: 'qiskit.ibmq'
+    :description: Allows integration with qiskit's hardware backends, and hardware-specific simulators.
+    :link: ibmq.html
+
+.. raw:: html
+
+        <div style='clear:both'></div>
+        </br>
+
+For example, the ``'qiskit.aer'`` device with two wires called like this:
+
+.. code-block:: python
+
+    import pennylane as qml
+    dev = qml.device('qiskit.aer', wires=2)
+
+Qiskit devices have different **backends**, which define which actual simulator or hardware is used by the
+device. Different simulator backends are optimized for different types of circuits. A backend can be defined as
+follows:
+
+.. code-block:: python
+
+    dev = qml.device('qiskit.aer', wires=2, backend='unitary_simulator')
+
+PennyLane chooses a default backend if no backend is specified.
+
+Tutorials
+~~~~~~~~~
+
+To see the PennyLane-Qiskit plugin in action, you can use any of the qubit based `demos
+from the PennyLane documentation <https://pennylane.ai/qml/demonstrations.html>`_, for example
+the tutorial on `qubit rotation <https://pennylane.ai/qml/demos/tutorial_qubit_rotation.html>`_,
+and simply replace ``'default.qubit'`` with any of the available Qiskit devices, such as ``'qiskit.aer'``:
+
+.. code-block:: python
+
+    dev = qml.device('qiskit.aer', wires=XXX)
+
+You can also try to run tutorials, such as the qubit rotation tutorial, on actual quantum hardware by
+using the ``'qiskit.ibmq'`` device.
+
+To filter tutorials that explicitly use a qiskit device, use the "Qiskit" filter on the right panel of the
+`demos <https://pennylane.ai/qml/demonstrations.html>`_.
```

Comment:
Nice! We should update it at some point, so that https://pennylane.ai/qml/demonstrations.html#qiskit automatically loads the page with the qiskit filter enabled.

Reply:
Yes, at the moment the filter does not seem to create a new linkable page...

Reply:
This will likely require some javascript on the demos.rst page, to parse any part of the URL after the `#` and use it to activate the filter

Reply:
OMG this is so cool!! :laughing: